contracts/src/token/erc721/extensions/metadata.rs

```diff
@@ -58,3 +61,24 @@ impl IErc721Metadata for Erc721Metadata {
         self._base_uri.get_string()
     }
 }
+
+impl IErc165 for Erc721Metadata {
```

Comment:
I am not sure if we want to have it implemented always.

Reply:
It is also [implemented](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/530179a71f435e85ae9df9b9f12b5637cf229e5c/contracts/token/ERC721/ERC721.sol#L50) for Erc721Metadata at solidity sources